src/test/kotlin/anttracker/ContactTest.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ContactTest.kt
+-----------------
+Revision History
+Rev. 1 - 2024/07/09 Original by M. Baker
+-------------------------------------------
+This file contains the tests for the function
+enterContactInformation from the Contact module.
+---------------------------------
+ */
+import anttracker.contact.enterContactInformation
+import anttracker.db.*
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.collections.shouldContainOnly
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import org.jetbrains.exposed.sql.transactions.transaction
+
+/**
+ * The goal of this test is to confirm that enterContactInformation correctly
+ * creates contacts, either for a singularly created contact, or multiple.
+ * Any contacts created must be inserted into the database.
+ */
+class ContactTest :
+    DescribeSpec({
+        xdescribe("enterContactInformation") {
+            describe("when a contact is created") {
```

Comment:
Since you are testing that the contact is created, that means `enterContactInformation` should never return `null` in this case. Therefore, I think throwing an exception would not make sense.

Reply:
I get build errors otherwise to make sure the return value isn't null :( What do you suggest?

Reply:
I threw an exception because enterContactInformation returning null isn't expected in this test case, and therefore it's "exceptional" behaviour.

Reply:
if `enterContactInformation` should not return `null`, than you would need to model controlling the user input so as to assure it does return a `Contact`.

Reply:
Which we can't do :(

Reply:
We can improve this test case for A5 if it's OK, because Russ did mention making tests in this way, depending on user input during the test. As you can remember he has a poor understanding of tests in the first place, as he referred to a validation test as a stress test 🤣